{
  "uid" : "d8f7af74cbe02aed",
  "name" : "Verify values of names and professions in specialist cards in the grid",
  "fullName" : "tests.specialists_page_test.TestSpecialistPageText#test_sp_02_04_verify_name_and_profession_in_cards",
  "historyId" : "78e03a13a1b27d1fe52cf41b87e50dab",
  "time" : {
    "start" : 1728608251187,
    "stop" : 1728608251187,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5623a9ac302a <unknown>\n#1 0x5623a97a95e0 <unknown>\n#2 0x5623a97a17eb <unknown>\n#3 0x5623a9791a09 <unknown>\n#4 0x5623a9793744 <unknown>\n#5 0x5623a9791cbd <unknown>\n#6 0x5623a979152f <unknown>\n#7 0x5623a9791422 <unknown>\n#8 0x5623a978f493 <unknown>\n#9 0x5623a978fada <unknown>\n#10 0x5623a97abf41 <unknown>\n#11 0x5623a983d8c5 <unknown>\n#12 0x5623a981db22 <unknown>\n#13 0x5623a983cd7d <unknown>\n#14 0x5623a981d8c3 <unknown>\n#15 0x5623a97eb6b3 <unknown>\n#16 0x5623a97ec68e <unknown>\n#17 0x5623a9a8da2b <unknown>\n#18 0x5623a9a919b1 <unknown>\n#19 0x5623a9a7a225 <unknown>\n#20 0x5623a9a92532 <unknown>\n#21 0x5623a9a5f38f <unknown>\n#22 0x5623a9ab1f28 <unknown>\n#23 0x5623a9ab20f3 <unknown>\n#24 0x5623a9ac1e7c <unknown>\n#25 0x7fe321a94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"969b533ec842bccc2dd3af097e06f02f\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fb25e17ef60>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5623a9ab20f3 \\\\u003Cunknown>\\\\n#24 0x5623a9ac1e7c \\\\u003Cunknown>\\\\n#25 0x7fe321a94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5623a9ac302a <unknown>\nE       #1 0x5623a97a95e0 <unknown>\nE       #2 0x5623a97a17eb <unknown>\nE       #3 0x5623a9791a09 <unknown>\nE       #4 0x5623a9793744 <unknown>\nE       #5 0x5623a9791cbd <unknown>\nE       #6 0x5623a979152f <unknown>\nE       #7 0x5623a9791422 <unknown>\nE       #8 0x5623a978f493 <unknown>\nE       #9 0x5623a978fada <unknown>\nE       #10 0x5623a97abf41 <unknown>\nE       #11 0x5623a983d8c5 <unknown>\nE       #12 0x5623a981db22 <unknown>\nE       #13 0x5623a983cd7d <unknown>\nE       #14 0x5623a981d8c3 <unknown>\nE       #15 0x5623a97eb6b3 <unknown>\nE       #16 0x5623a97ec68e <unknown>\nE       #17 0x5623a9a8da2b <unknown>\nE       #18 0x5623a9a919b1 <unknown>\nE       #19 0x5623a9a7a225 <unknown>\nE       #20 0x5623a9a92532 <unknown>\nE       #21 0x5623a9a5f38f <unknown>\nE       #22 0x5623a9ab1f28 <unknown>\nE       #23 0x5623a9ab20f3 <unknown>\nE       #24 0x5623a9ac1e7c <unknown>\nE       #25 0x7fe321a94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728608251187,
      "stop" : 1728608251960,
      "duration" : 773
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728608130626,
      "stop" : 1728608130626,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728608251960,
      "stop" : 1728608252446,
      "duration" : 486
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5623a9ac302a <unknown>\n#1 0x5623a97a95e0 <unknown>\n#2 0x5623a97a17eb <unknown>\n#3 0x5623a9791a09 <unknown>\n#4 0x5623a9793744 <unknown>\n#5 0x5623a9791cbd <unknown>\n#6 0x5623a979152f <unknown>\n#7 0x5623a9791422 <unknown>\n#8 0x5623a978f493 <unknown>\n#9 0x5623a978fada <unknown>\n#10 0x5623a97abf41 <unknown>\n#11 0x5623a983d8c5 <unknown>\n#12 0x5623a981db22 <unknown>\n#13 0x5623a983cd7d <unknown>\n#14 0x5623a981d8c3 <unknown>\n#15 0x5623a97eb6b3 <unknown>\n#16 0x5623a97ec68e <unknown>\n#17 0x5623a9a8da2b <unknown>\n#18 0x5623a9a919b1 <unknown>\n#19 0x5623a9a7a225 <unknown>\n#20 0x5623a9a92532 <unknown>\n#21 0x5623a9a5f38f <unknown>\n#22 0x5623a9ab1f28 <unknown>\n#23 0x5623a9ab20f3 <unknown>\n#24 0x5623a9ac1e7c <unknown>\n#25 0x7fe321a94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728608251960,
        "stop" : 1728608252446,
        "duration" : 486
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5623a9ac302a <unknown>\n#1 0x5623a97a95e0 <unknown>\n#2 0x5623a97a17eb <unknown>\n#3 0x5623a9791a09 <unknown>\n#4 0x5623a9793744 <unknown>\n#5 0x5623a9791cbd <unknown>\n#6 0x5623a979152f <unknown>\n#7 0x5623a9791422 <unknown>\n#8 0x5623a978f493 <unknown>\n#9 0x5623a978fada <unknown>\n#10 0x5623a97abf41 <unknown>\n#11 0x5623a983d8c5 <unknown>\n#12 0x5623a981db22 <unknown>\n#13 0x5623a983cd7d <unknown>\n#14 0x5623a981d8c3 <unknown>\n#15 0x5623a97eb6b3 <unknown>\n#16 0x5623a97ec68e <unknown>\n#17 0x5623a9a8da2b <unknown>\n#18 0x5623a9a919b1 <unknown>\n#19 0x5623a9a7a225 <unknown>\n#20 0x5623a9a92532 <unknown>\n#21 0x5623a9a5f38f <unknown>\n#22 0x5623a9ab1f28 <unknown>\n#23 0x5623a9ab20f3 <unknown>\n#24 0x5623a9ac1e7c <unknown>\n#25 0x7fe321a94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"969b533ec842bccc2dd3af097e06f02f\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728608252479,
      "stop" : 1728608252541,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Specialists Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "specialists_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestSpecialistsPage"
  }, {
    "name" : "host",
    "value" : "fv-az523-260"
  }, {
    "name" : "thread",
    "value" : "2358-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.specialists_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 8,
        "broken" : 5,
        "skipped" : 7,
        "passed" : 333,
        "unknown" : 0,
        "total" : 353
      },
      "items" : [ {
        "uid" : "de1210f9066d2f53",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/de1210f9066d2f53",
        "status" : "passed",
        "time" : {
          "start" : 1728557352793,
          "stop" : 1728557352905,
          "duration" : 112
        }
      }, {
        "uid" : "6ba51609318bcc50",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/6ba51609318bcc50",
        "status" : "passed",
        "time" : {
          "start" : 1728554225007,
          "stop" : 1728554225114,
          "duration" : 107
        }
      }, {
        "uid" : "94dc658126d6fda5",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/94dc658126d6fda5",
        "status" : "passed",
        "time" : {
          "start" : 1728522226066,
          "stop" : 1728522226174,
          "duration" : 108
        }
      }, {
        "uid" : "42bd9e022fd30d51",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/42bd9e022fd30d51",
        "status" : "passed",
        "time" : {
          "start" : 1728470627304,
          "stop" : 1728470627414,
          "duration" : 110
        }
      }, {
        "uid" : "80cb426afea98c97",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/80cb426afea98c97",
        "status" : "passed",
        "time" : {
          "start" : 1728468053249,
          "stop" : 1728468053370,
          "duration" : 121
        }
      }, {
        "uid" : "c2635e3c050382d2",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/c2635e3c050382d2",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728435895178,
          "stop" : 1728435895178,
          "duration" : 0
        }
      }, {
        "uid" : "a0c2e6a0303fbdbc",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/a0c2e6a0303fbdbc",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728378104984,
          "stop" : 1728378104984,
          "duration" : 0
        }
      }, {
        "uid" : "f5f1ecbfa3c17c4d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/f5f1ecbfa3c17c4d",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728375313661,
          "stop" : 1728375313661,
          "duration" : 0
        }
      }, {
        "uid" : "2eff99d6e1f3d383",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/2eff99d6e1f3d383",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728375195224,
          "stop" : 1728375195224,
          "duration" : 0
        }
      }, {
        "uid" : "809af479ae923368",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/809af479ae923368",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728349283955,
          "stop" : 1728349283955,
          "duration" : 0
        }
      }, {
        "uid" : "207f24cd5ce14748",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/207f24cd5ce14748",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728293033995,
          "stop" : 1728293033995,
          "duration" : 0
        }
      }, {
        "uid" : "3b2ea38f3883c000",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/3b2ea38f3883c000",
        "status" : "skipped",
        "statusDetails" : "Skipped: unsupported preconditions",
        "time" : {
          "start" : 1728290083566,
          "stop" : 1728290083566,
          "duration" : 0
        }
      }, {
        "uid" : "3212a3e587188bfe",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/3212a3e587188bfe",
        "status" : "failed",
        "statusDetails" : "AssertionError: Names in specialist cards mismatch the valid values\nassert ['Login'] in [['Гарбарук Екатерина Сергеевна', 'Прошина Любовь Александровна', 'Березкина Ксения Сергеевна', 'Метельская Наталья Николаевна', 'Королева Инна Васильевна', 'Юлия Кибалова', ...], ['Ekaterina Garbaruk', 'Lubov Proshina', 'Ksenia Berezkina', 'Natalia Metelskaya', 'Inna Koroleva', 'Yulia Kibalova', ...]]\n +  where [['Гарбарук Екатерина Сергеевна', 'Прошина Любовь Александровна', 'Березкина Ксения Сергеевна', 'Метельская Наталья Николаевна', 'Королева Инна Васильевна', 'Юлия Кибалова', ...], ['Ekaterina Garbaruk', 'Lubov Proshina', 'Ksenia Berezkina', 'Natalia Metelskaya', 'Inna Koroleva', 'Yulia Kibalova', ...]] = SpecialistsPageData.specialists_names",
        "time" : {
          "start" : 1728263560887,
          "stop" : 1728263560931,
          "duration" : 44
        }
      }, {
        "uid" : "55d9d9a4b0ce8650",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/55d9d9a4b0ce8650",
        "status" : "failed",
        "statusDetails" : "AssertionError: Names in specialist cards mismatch the valid values\nassert ['Login'] in [['Гарбарук Екатерина Сергеевна', 'Прошина Любовь Александровна', 'Березкина Ксения Сергеевна', 'Метельская Наталья Николаевна', 'Королева Инна Васильевна', 'Юлия Кибалова', ...], ['Ekaterina Garbaruk', 'Lubov Proshina', 'Ksenia Berezkina', 'Natalia Metelskaya', 'Inna Koroleva', 'Yulia Kibalova', ...]]\n +  where [['Гарбарук Екатерина Сергеевна', 'Прошина Любовь Александровна', 'Березкина Ксения Сергеевна', 'Метельская Наталья Николаевна', 'Королева Инна Васильевна', 'Юлия Кибалова', ...], ['Ekaterina Garbaruk', 'Lubov Proshina', 'Ksenia Berezkina', 'Natalia Metelskaya', 'Inna Koroleva', 'Yulia Kibalova', ...]] = SpecialistsPageData.specialists_names",
        "time" : {
          "start" : 1728226726880,
          "stop" : 1728226726925,
          "duration" : 45
        }
      }, {
        "uid" : "9fb0bbc1f4b51deb",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/9fb0bbc1f4b51deb",
        "status" : "failed",
        "statusDetails" : "AssertionError: Names in specialist cards mismatch the valid values\nassert ['Login'] in [['Гарбарук Екатерина Сергеевна', 'Прошина Любовь Александровна', 'Березкина Ксения Сергеевна', 'Метельская Наталья Николаевна', 'Королева Инна Васильевна', 'Юлия Кибалова', ...], ['Ekaterina Garbaruk', 'Lubov Proshina', 'Ksenia Berezkina', 'Natalia Metelskaya', 'Inna Koroleva', 'Yulia Kibalova', ...]]\n +  where [['Гарбарук Екатерина Сергеевна', 'Прошина Любовь Александровна', 'Березкина Ксения Сергеевна', 'Метельская Наталья Николаевна', 'Королева Инна Васильевна', 'Юлия Кибалова', ...], ['Ekaterina Garbaruk', 'Lubov Proshina', 'Ksenia Berezkina', 'Natalia Metelskaya', 'Inna Koroleva', 'Yulia Kibalova', ...]] = SpecialistsPageData.specialists_names",
        "time" : {
          "start" : 1728221827196,
          "stop" : 1728221827243,
          "duration" : 47
        }
      }, {
        "uid" : "88861c16cd2bf1f4",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/88861c16cd2bf1f4",
        "status" : "passed",
        "time" : {
          "start" : 1728177058422,
          "stop" : 1728177058547,
          "duration" : 125
        }
      }, {
        "uid" : "1488afdcda77a058",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/1488afdcda77a058",
        "status" : "passed",
        "time" : {
          "start" : 1728117691865,
          "stop" : 1728117691977,
          "duration" : 112
        }
      }, {
        "uid" : "1769a400cc67afec",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/1769a400cc67afec",
        "status" : "passed",
        "time" : {
          "start" : 1728109071205,
          "stop" : 1728109071314,
          "duration" : 109
        }
      }, {
        "uid" : "a74ae0aeb868bbb1",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/143//#testresult/a74ae0aeb868bbb1",
        "status" : "passed",
        "time" : {
          "start" : 1728090329011,
          "stop" : 1728090329125,
          "duration" : 114
        }
      }, {
        "uid" : "15858200119189b9",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/142//#testresult/15858200119189b9",
        "status" : "passed",
        "time" : {
          "start" : 1728003948785,
          "stop" : 1728003948903,
          "duration" : 118
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "d8f7af74cbe02aed.json",
  "parameterValues" : [ ]
}